cluster-operator/src/test/java/io/strimzi/operator/cluster/model/KafkaBridgeClusterTest.java

```diff
@@ -970,44 +970,6 @@ public void testNullClusterRoleBinding() {
         assertThat(crb, is(nullValue()));
     }
 
-    @ParallelTest
-    public void testKafkaBridgeContainerEnvVarsConflict() {
```

Comment:
We still have some env vars, so why not update it instead of deleting it?

Reply:
I brought it back changing the env vars to be used in the check.